SAMv1.tex

```diff
@@ -658,6 +659,19 @@ \section{Recommended Practice for the SAM Format}
     should be present.
   \item The {\tt NM} tag should be present.
   \end{enumerate}
+\item Alignment to circular reference sequences
+  This applies to circular reference sequences that are annotated with
+  {\tt TP:circular} in the {\tt @SQ} header line.
+  \begin{enumerate}[label=\arabic*]
+  \item (preferred) {\sf POS} should be between 0 and the length specified 
+    in {\tt LN} field of the {\tt @SQ} header line with an {\tt SN} equal to
+    {\sf RNAME}. {\sf POS} plus the sum of lengths of {\tt M/=/X/D/N} 
+    operations in {\sf CIGAR} may exceed the length specified in the {\tt LN}
+    field of the {\tt @SQ} header line (if exists) with an SN equal to 
+    {\sf RNAME}.
+  \item (alternative) The alignment may be split across multiple records,
```

Comment:
here we should explicitly say that one of the record should "start" at POS=1 and another should "end" at POS=LN. otherwise, we are not really saying much in the case.

Reply:
What about a CDS where the origin is in an intron? In that case a GenBank/EMBL style CDS record would be multi-p